assets/src/domain/index.ts

```diff
@@ -0,0 +1,14 @@
+import './shared/services/publicPath';
+
+const domain = window?.eeDomain;
+
+if (domain) {
+	import(
+		/* webpackExclude: /(shared)/ */
+		/* webpackChunkName: "[request]" */
+		`./${domain}/main.ts`
+	).catch(console.error);
+	console.log('domain', domain);
```

Comment:
what about:
```
	import(
		/* webpackExclude: /(shared)/ */
		/* webpackChunkName: "[request]" */
		`./${domain}/main.ts`
	)
.then(() => {
	console.log('domain', domain);
})
.catch(console.error);
```
?
it is a bit uncommon to see a promise with just a catch

Reply:
Done